bundles/core/src/main/java/com/adobe/cq/commerce/core/components/services/urls/package-info.java

```diff
@@ -13,7 +13,7 @@
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
-@Version("1.2.0")
+@Version("2.0.0")
```

Comment:
This should not be necessary.

Reply:
does not build without it. because I changed a method signature